master/static/migrations/20231204112957_run_id_task_id.tx.up.sql

```diff
@@ -0,0 +1,11 @@
+-- Swap checkpoints v2 constraint over to tasks like it should have been.
+ALTER TABLE checkpoints_v2 DROP CONSTRAINT checkpoints_v2_task_id_fkey;
+ALTER TABLE checkpoints_v2 ADD CONSTRAINT checkpoints_v2_task_id_fkey
+  FOREIGN KEY (task_id) REFERENCES tasks(task_id) ON DELETE CASCADE;
```

Comment:
is this just to make it not-null delete cascade?

there is a comment on the initial migration, if it adds any context
> -- It is possible a checkpoint may be imported, in which case it would have no task or
> -- allocation ID. We make it null (i fixed a typo here) now to discourage relying on it so that feature is easier.

Reply:
needed to switch the constraint back to tasks

since now tasks to trials is a MTM which means we can't FK to it